src/node-lib/curriculum-api-2023/queries/pupilLessonOverview/pupilLessonOverview.gql

```diff
@@ -23,11 +23,10 @@ query pupilLessonOverview(
     supervisionLevel
     contentGuidance
     lessonEquipmentAndResources
-    worksheetUrl
     videoTitle
     videoMuxPlaybackId
     videoWithSignLanguageMuxPlaybackId
     transcriptSentences
-    lessonCohort
+    isLegacy
```

Comment:
If i'm following the intention of this new field it is equivalent to `lesson.lessonCohort === LEGACY_COHORT` which means we pull this boolean out ourselves and use it directly.